src/main/java/it/gov/pagopa/payhub/activities/connector/debtposition/client/TransferSearchClient.java

```diff
@@ -17,10 +19,10 @@ public TransferSearchClient(DebtPositionApisHolder debtPositionApisHolder) {
         this.debtPositionApisHolder = debtPositionApisHolder;
     }
 
-    public Transfer findBySemanticKey(Long orgId, String iuv, String iur, Integer transferIndex, String accessToken) {
+    public Transfer findBySemanticKey(Long orgId, String iuv, String iur, Integer transferIndex, Set<String> installmentStatusSet, String accessToken) {
```

Comment:
let's use the right type until Client, the Client should hide the conversion to String (due to queryParam nature)